webapp/src/components/table/table.tsx

```diff
@@ -26,6 +29,25 @@ type Props = {
     onCardClicked: (e: React.MouseEvent, card: Card) => void
 }
 
+// Eventually, these will need to come from the theme.
+// Currently, the theme doesn't change any padding or font descriptor
+const header = {
```

Comment:
Do we can extract this information from the `TableHeader` component using the existing content? probably we can calculate it better there, and we can make it more generic here, so... the table Header give you the template id, and the new size, and you don't have to figure out every single use case here.

Reply:
Sorry, I'm not sure I understand which part to move to `TableHeader`. Just the calculation of the header width, right.

Reply:
Yes, the thing is, you are taking the value, exposing the font size of the element, and all that, but in the TableHeader you probably already have much of the information inside, and you have all you need to make the change. So instead of passing directly the "onAutoSizeColumn" to the HorizontalGrip, you can have a function in the middle that populates the new width data base on the current title element (you can do data with a ref, and some javascript to get  styles/font-size/font-type) or maybe there is a better way, I don't know, but problably is the place to get that info.